gallery.php

```diff
@@ -0,0 +1,24 @@
+<?php
+include "functions.php";
```

Comment:
include will let you require a file that doesn't exist, but if `functions.php` doesn't exist that's a problem! Use `require` instead